src/commands/assign.js

```diff
@@ -207,7 +235,11 @@ async function checkAssigned() {
         }
         // Add the id of the newly assigned submission to the list of assigned
         // submissions.
-        assigned.push(s.id);
+        assigned.push({
+          submission_id: s.id,
```

Comment:
Ok. It dind't complain it. I was using camlCase but then I saw this 

```
    unreadFeedbacks.slice(-diff).forEach((fb) => {
      feedbackNotification(fb.rating, fb.project.name, fb.submission_id);
    });
```

So I preferred to change. :smile: